src/Symfony/Component/Notifier/Bridge/Telegram/Reply/Markup/ReplyKeyboardMarkup.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Telegram\Reply\Markup;
+
+use Symfony\Component\Notifier\Bridge\Telegram\Reply\Markup\Button\KeyboardButton;
+
+/**
+ * @author Mihail Krasilnikov <mihail.krasilnikov.j@gmail.com>
+ *
+ * @see https://core.telegram.org/bots/api#replykeyboardmarkup
+ */
+class ReplyKeyboardMarkup extends AbstractTelegramReplyMarkup
+{
+    public function __construct()
+    {
+        $this->options['keyboard'] = [];
+    }
+
+    public function keyboard(array $buttons): self
```

Comment:
Please provide phpdoc for describe `$buttons` type, like `@param KeyboardButton[] $buttons`

Reply:
Added